### ISSUE TITLE
Allow user to correct the order of circles detected for the exported CSV

### ISSUE THREAD

**Issue by digitaldias** [digitaldias/DropSampler#1]

**Comment by TessFerrandez** [comment]
Not sure if this is needed anymore.... the order of the circles in the CSV will be by row/column

**Comment by digitaldias** [comment]
Some of the images appear 90 degrees rotated. Should we demand a specific layout?

**Comment by TessFerrandez** [comment]
Yes, for sure... I will demand both that they are in the right direction... and that all of them are lined up one after another.

**Comment by peterbryntesson** [comment]
This is already supported in feature/add-missing-circles I think
